packages/transport/test/modules/sync/SyncController.ordered.test.ts

```diff
@@ -0,0 +1,48 @@
+import { IDatabaseConnection } from "@js-soft/docdb-access-abstractions";
+import { CoreDate } from "@nmshd/core-types";
+import { AccountController, Transport } from "../../../src";
+import { TestUtil } from "../../testHelpers/TestUtil";
+
+describe("SyncController.ordered", function () {
+    let connection: IDatabaseConnection;
+    let transport: Transport;
+
+    let sender: AccountController | undefined;
+    let recipient: AccountController | undefined;
+    let recipientSecondDevice: AccountController | undefined;
+
+    beforeAll(async function () {
+        connection = await TestUtil.createDatabaseConnection();
+
+        transport = TestUtil.createTransport(connection, { datawalletEnabled: true });
+        await transport.init();
+
+        sender = await TestUtil.createAccount(transport);
+        recipient = await TestUtil.createAccount(transport);
+    });
+
+    afterAll(async () => {
+        await sender?.close();
+        await recipient?.close();
+        await recipientSecondDevice?.close();
+
+        await connection.close();
+    });
+
+    // eslint-disable-next-line jest/expect-expect
+    test("should execute datawallet modifications in the correct order", async function () {
```

Comment:
I don't understand how the current implementation of this test should assert that the datawallet modifications are executed in the correct order. Is the fact that no exception is thrown  sufficient for you?

Reply:
I have added a comment